.github/workflows/nucypher-core.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.target }}
           override: true
-      - run: cargo install wasm-pack
+      - run: cargo install wasm-opt wasm-pack
```

Comment:
For some reason, `wasm-pack` had issues finding `wasm-opt` here. This was supposed to be [fixed](https://github.com/rustwasm/wasm-pack/releases/tag/v0.11.1) in the last release, but for some reason didn't work for us here.